<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>not_rentedboxwindow</class>
 <widget class="QMainWindow" name="not_rentedboxwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>456</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj najemcę</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rozmiar boxu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPowr_t">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionPowr_t"/>
   </widget>
   <addaction name="menuPowr_t"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPowr_t">
   <property name="text">
    <string>Powrót</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
